src/constants/index.tsx

```diff
@@ -0,0 +1,5 @@
+// as per @govtechsg/open-attestation/dist/types/__generated__/schema.2.0 and schema.3.0 , but omit `Did = "DID"`
+export enum IdentityProofType {
+  DNSDid = "DNS-DID",
+  DNSTxt = "DNS-TXT",
+}
```

Comment:
since u change this, should it all be in caps? like DNSDID & DNSTXT

Reply:
nope, following from oa generated types

Reply:
ok

Reply:
to be fair i prefer the previous naming `DnsDid`, but since oa side is not, is either we:
- change oa side = risk that other downstream projects need to update later
- follow what was there already <- prefer this